.gitignore

```diff
@@ -1,33 +1,34 @@
-*~
-*.gch*
-*.obj
-*.o
-*.s
-*.exe
-Factor/factor
+*#*#
+*.*.marks
+*.RES
 *.a
+*.bak
 *.dll
-*.lib
+*.dylib
+*.exe
 *.exp
-*.res
-*.RES
+*.gch*
 *.image
-factor.image.fresh
-*.dylib
-factor.com
-*#*#
-.DS_Store
-.gdb_history
-*.*.marks
+*.lib
+*.o
+*.obj
+*.res
+*.s
+*.so
+*~
+.#*
 .*.swm
 .*.swn
 .*.swo
 .*.swp
+.DS_Store
+.gdb_history
+/factor
+Factor.app/Contents/MacOS/factor
+Factor.app/Contents/_CodeSignature
+a.out
+checksums.txt
+factor.com
+factor.image.fresh
 logs
```

Comment:
I *suspect* we may want this and the following line to begin a `/` so that they only apply in the repo root, but I didn't fold that into this PR.

Reply:
I meant to say *this* makes sense -- apply only at repo root.

Reply:
I'll update the PR with what I think we're agreeing on and then do a sanity check before merging. :)